<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TP2_QtClass</class>
 <widget class="QMainWindow" name="TP2_QtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TP2_Qt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="CB_Ports">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_OpenPort">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ouvrir le port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Status port :</string>
    </property>
   </widget>
   <widget class="QLabel" name="TXT_Status">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fermé</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Données reçues :</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="RECEIVEDTXT_Data">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>551</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TP2_Qt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>BTN_OpenPort</sender>
   <signal>clicked()</signal>
   <receiver>TP2_QtClass</receiver>
   <slot>onDisplayBTN_OpenPortClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onDisplayBTN_OpenPortClicked()</slot>
 </slots>
</ui>
